akka-docs/rst/scala/logging.rst

```diff
@@ -359,6 +364,18 @@ the first case and ``LoggerFactory.getLogger(s: String)`` in the second).
 
   val log = Logging(system.eventStream, "my.nice.string")
 
+.. _slf4j-directly-scala:
+
+Using the SLF4J API directly
+----------------------------
+If you use the SLF4J API directly in your application, remember that log operations will block
+while logging implementation behind it writing the log entries to disk.
```

Comment:
grammar a bit weird here, let's

`... the logging operations will block while the underlying infrastructure writes the log statements.`